<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filtrar_venci</class>
 <widget class="QDialog" name="filtrar_venci">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filtrar vencimientos para domiciliación</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>120</height>
        </size>
       </property>
       <property name="title">
        <string>Intervalo importes</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="imp_desdelineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="imp_hastalineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>120</height>
        </size>
       </property>
       <property name="title">
        <string>Intervalo cuentas</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="cta_desdelineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="cta_hastalineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="vencigroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>120</height>
        </size>
       </property>
       <property name="title">
        <string>Fecha vencimiento</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QDateEdit" name="venci_desdedateEdit"/>
        </item>
        <item>
         <widget class="QDateEdit" name="venci_hastadateEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="fecha_opgroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>120</height>
        </size>
       </property>
       <property name="title">
        <string>Fecha operación</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QDateEdit" name="op_desdedateEdit"/>
        </item>
        <item>
         <widget class="QDateEdit" name="op_hastadateEdit"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="todospushButton">
       <property name="text">
        <string>&amp;Todos</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/reload2.png</normaloff>:/iconos/graf/reload2.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ningunopushButton">
       <property name="text">
        <string>&amp;Ninguno</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/cruz.png</normaloff>:/iconos/graf/cruz.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ordenarpushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/sort_ascend.png</normaloff>:/iconos/graf/sort_ascend.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cuenta tesorería:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="aux_bancolineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="codigopushButton">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="borracodigopushButton">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Eliminar asignación</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/borraelem.png</normaloff>:/iconos/graf/borraelem.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+L</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="columnCount">
      <number>10</number>
     </property>
     <column>
      <property name="text">
       <string>Sel</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Número</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Concepto</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha op.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Importe</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Vencimiento</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cuenta</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Descrip Cta</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Documento</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Externo</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refrescarpushButton">
       <property name="text">
        <string>Refrescar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/refresh.xpm</normaloff>:/iconos/graf/refresh.xpm</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelarpushButton">
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/borrar.png</normaloff>:/iconos/graf/borrar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptarpushButton">
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/cheq.png</normaloff>:/iconos/graf/cheq.png</iconset>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>imp_desdelineEdit</tabstop>
  <tabstop>imp_hastalineEdit</tabstop>
  <tabstop>cta_desdelineEdit</tabstop>
  <tabstop>cta_hastalineEdit</tabstop>
  <tabstop>vencigroupBox</tabstop>
  <tabstop>venci_desdedateEdit</tabstop>
  <tabstop>venci_hastadateEdit</tabstop>
  <tabstop>fecha_opgroupBox</tabstop>
  <tabstop>op_desdedateEdit</tabstop>
  <tabstop>op_hastadateEdit</tabstop>
  <tabstop>todospushButton</tabstop>
  <tabstop>ningunopushButton</tabstop>
  <tabstop>aux_bancolineEdit</tabstop>
  <tabstop>codigopushButton</tabstop>
  <tabstop>borracodigopushButton</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>refrescarpushButton</tabstop>
  <tabstop>cancelarpushButton</tabstop>
  <tabstop>aceptarpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>filtrar_venci</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aceptarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>filtrar_venci</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>664</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
